src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -76,15 +78,18 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     String user = userService.getCurrentUser().getEmail();
-    String userEnteredContent = request.getParameter("text");
-    
+    String test = request.getParameter("text");
```

Comment:
I think you can clean this code up a little bit by removing the test variable. You might have been playing around with the code and left it here accidentally. It works well, just send a new commit that fixes this very slight issue. Other than that it looks nice and it compiles!

Reply:
Yes, I forgot about that, It is fixed now. Thanks